tools/ci_build/github/linux/docker/inference/x86_64/default/cpu/scripts/install_deps.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
This should be placed under tools\ci_build\github\linux\docker\scripts\inference just like tools\ci_build\github\linux\docker\scripts\training. The docker folder contains dockerfiles and the scripts folder only.

Reply:
The dockerfile can't access the  tools\ci_build\github\linux\docker\scripts\inference folder. Because a docker file can only access files in the same folder or child folders.

Reply:
Then should the Dockerfile be moved up to the tools\ci_build\github\linux\docker folder? Would that be more consistent with the location of the rest of the Dockerfile*'s?

Reply:
The consistence is the thing I'm trying to avoid and break. We used to share the scripts across different docker images and pipelines. But as the project grows, it becomes not manageable. Everyone wants to add new dependency there, and they add it for every pipeline. For example, pytorch was added to inference pipelines just because the author who made the change didn't know how to do it for training only. But the consequence is we can't add arm support to inferences pipelines: pytorch is too big to be fit for ARM.  So here I need to create a new folder and start over doing the things in the new place, without having the unnecessary dependencies.

Reply:
Alright, let's get this out for 1.9 release and then see how we can do better.